paasta_tools/paasta_maintenance.py

```diff
@@ -422,6 +424,43 @@ def status():
     print "%s" % status.text
 
 
+def is_safe_to_kill(hostname):
+    """Checks if a host has drained or reached its maintenance window
+    :param hostname: hostname to check
+    :returns: True or False
+    """
+    if is_host_drained(hostname) or hostname in get_hosts_in_maintenance_window():
```

Comment:
`return is_host_drained(hostname) or hostname in get_hosts_in_maintenance_window()`